src/components/WhyPlayParkour/WhyPlayParkour.jsx

```diff
@@ -0,0 +1,18 @@
+import './WhyPlayParkour.css';
+import { useTranslation } from 'react-i18next'; 
+import { CarouselComponent } from '../Carousel/Carousel';
+
+export const WhyPlayParkour = () => {
+
+  const { t } = useTranslation(); 
+
+  return (
+    <div className="whyplay-wrapper">
+      <h1>{t('WhyPlay.heading')}</h1> 
+      <div className="icons-wrapper">
+        <CarouselComponent />
```

Comment:
This does not display on larger screen sizes. It seems it is pushed very far to the right and needs to be pulled in by the user.